packages/suite-desktop/src-electron/app.ts

```diff
@@ -83,6 +83,17 @@ const init = async () => {
     logger.debug('init', `Load URL (${src})`);
     mainWindow.loadURL(src);
 
+    let resizeDebounce: Timeout | null = null;
+    mainWindow.on('resize', () => {
+        if (resizeDebounce) return;
+        resizeDebounce = setTimeout(() => {
+            resizeDebounce = null;
+            const winBound = mainWindow.getBounds() as WinBounds;
+            store.setWinBounds(winBound);
+            logger.debug('app', 'new winBounds saved');
+        }, 1000);
+    });
+
```

Comment:
not sure if debounce is necessary. `resize` event does not fire so often. but maybe it is "better" to do some debouncing here.

Reply:
I believe debounce is a good idea here, but you have to check if `mainWindow` is still defined before trying to getBounds() and would be nice to clear that timeout on `closed`

Reply:
added [ca43b7a](https://github.com/trezor/trezor-suite/pull/4931/commits/ca43b7ad5e56f29b157ed50c7e0b3da7f72082e9)